src/app/plugins/scores.plugin.ts

```diff
@@ -148,10 +153,25 @@ export class ScoresPlugin extends Plugin {
       this.container.loggerService.error(error);
     }
   }
-  private _tryUpcomingGame(embedBuffer: RichEmbed[], game: any, upcomingRegex: RegExp, upcomingTimeRegex: RegExp, teamArg: string): boolean {
 
-    let teamsData = game.match(upcomingRegex);
-    const time = String(game.match(upcomingTimeRegex)); //.split does not work unless you cast string
+  private _getHelp(message: IMessage) {
+    message.reply({
+      embed: {
+        title: 'Scores Plugin Help',
+        color: '3447003',
+        fields: [
+          { name: 'Supported Sports', value: 'NCAA NFL MLB NBA' },
+          { name: 'Inputting Teams', value: 'For NCAA: Use their most common name\nFor Professional Sports: Use their city only\n' },
+          { name: 'Examples', value: '!scores NCAA UCF\n!scores NCAA Florida State\n!scores NFL Seattle\n!scores MLB Atlanta' }
+        ]
+      }
+    });
+  }
+
+  private _tryUpcomingGame(embedBuffer: RichEmbed[], game: any, teamArg: string): boolean {
+
+    let teamsData = game.match(this._UPCOMING_REGEX);
+    const time = String(game.match(this._UPCOMING_TIME_REGEX)); //.split does not work unless you cast string
```

Comment:
why not access .match()[0] ?

Reply:
if u do what corgo suggests, don't directly use indexes!! use array destructuring :)

Reply:
I don't access [0] because game.match(reg) can be null. I check for that first then access when processing the data

Reply:
If you do `String(null)`, it will return a string literal of `'null'` is this desired? if not, you can do `const time = game.match(this._UPCOMING_TIME_REGEX) || '').split(...)`

Reply:
I don't want to continue if it is null, so that's why I don't access until `if(!teamsData || !time){ return false; }`
If the matches return null that means either the team requested A) Isn't playing this week, or B) Doesn't exist.
So it is helpful to know when it is null rather than set it to empty string and go on